front/app/modules/commercial/content_builder/admin/components/contentBuilderToggle.test.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { render, screen } from 'utils/testUtils/rtl';
+import ContentBuilderToggle from './contentBuilderToggle';
+import { WithRouterProps } from 'react-router';
+
+jest.mock('services/locale');
+jest.mock('services/avatars');
+jest.mock('utils/cl-intl');
+jest.mock('services/appConfiguration');
+jest.mock('utils/cl-router/history');
+jest.mock('components/UI/QuillEditor/QuillMultilocWithLocaleSwitcher');
+
+const getRouterProps = (projectId) =>
+  ({
+    location: {
+      pathname: '/admin/projects/projectID/description',
+    },
+    params: {
+      projectId,
+    },
+  } as any as WithRouterProps);
+
+const dummyOnChange = () => {};
+
+const routerProps = getRouterProps('projectID');
+
+describe('ContentBuilderToggle', () => {
+  it('Confirm link shown appropriately when builder option toggled', () => {
```

Comment:
Nice! It's a good idea to test both states - when the toggle is enabled it should link and the description is not in the doc - when disabled - the opposite. 

Also a tip - you can mock `react-router` directly like so:
```jest.mock('react-router', () => {
  return {
    withRouter: (Component) => {
      return (props) => {
        return <Component {...props} params={{ viewId }} />;
      };
    },
    Link: 'Link',
  };
});
```

Reply:
@IvaKop Do you know if there is an example where we have mocked/tested a QuillMultilocWithLocaleSwitcher editor? It doesn't appear in the Render output, however I know it's being hidden/shown correctly since I tested with a temporary h1 element in the same code block.

Reply:
Looks like you need to mock `useLocale`, otherwise the component will return null.